Common/FeatureFlags.swift

```diff
@@ -114,10 +116,24 @@ struct FeatureFlagConfiguration: Decodable {
         #endif
 
         // Swift compiler config is inverse, since the default state is enabled.
-        #if WALSH_INSULIN_MODEL_DISABLED
-        self.walshInsulinModelEnabled = false
+        #if SIRI_DISABLED
+        self.siriEnabled = false
         #else
-        self.walshInsulinModelEnabled = true
+        self.siriEnabled = true
+        #endif
+        
+        // Swift compiler config is inverse, since the default state is enabled.
+        #if AUTOMATIC_BOLUS_DISABLED
+        self.automaticBolusEnabled = false
+        #else
+        self.automaticBolusEnabled = true
+        #endif
+        
+        // Swift compiler config is inverse, since the default state is enabled.
+        #if OUTSIDE_DOSES_DISABLED
```

Comment:
Shouldn't the default state be `disabled` for this?

Reply:
Not sure why we'd want default to be disabled.

Reply:
See above on preset activation.